demonstrations/tutorial_rl_pulse.py

```diff
@@ -0,0 +1,1049 @@
+r"""
+Gate calibration with reinforcement learning
+============================================
+
+Gate-based quantum circuits are the most common representation of quantum computations. These
+provide an abstraction layer that enables the development of quantum algorithms without considering
+the hardware in charge of the execution. However, every quantum platform offers a different set of
+interactions and controls that define the natural operations that can be performed in the hardware.
+These are known as the *native gates* of the device, and they constitute the fundamental
+building blocks of any quantum algorithm executed in it. Therefore, it is essential that such
+operations are performed as accurately as possible, which requires the careful tunning of the
+hardware's controls. In this demo, we will learn how to use reinforcement learning to find the
+optimal control parameters to accurately execute quantum gates. We will implement an
+experimentally-friendly protocol based on the direct interaction with the hardware, following
+the main ideas in [#BaumPRXQ21]_, which we illustrate using superconducting quantum computers as
+an example.
```

Comment:
```suggestion
the main ideas in [#BaumPRXQ21]_, which we illustrate using superconducting qubits as
an example.
```

This may sound like we are doing stuff experimentally, while what you mean is we look at superconducting qubit platforms I think